CANdy/test/domain_transactions_genesis

```diff
@@ -0,0 +1,8 @@
+{"reqSignature":{},"txn":{"data":{"alias":"Wade Barnes","dest":"MNHGnsZF4KS6wykfRKUuLS","role":"0","verkey":"C6qcDRQiswUGpWK4FgDsHtaVUVjR4nRDYurZAivoetPZ"},"metadata":{},"type":"1"},"txnMetadata":{"seqNo":1},"ver":"1"}
+{"reqSignature":{},"txn":{"data":{"alias":"Stephen Curran","dest":"7aMWWD5d6dyb9FHY6RisAn","role":"0","verkey":"~Wg2CDurCSNyVfMypkK66Xz"},"metadata":{},"type":"1"},"txnMetadata":{"seqNo":2},"ver":"1"}
+{"reqSignature":{},"txn":{"data":{"alias":"Clecio Varjao","dest":"HL7vUSZi2fKzG3WsUo1CBf","role":"0","verkey":"~PJKwjd44UBHQ7KNc9Wtqav"},"metadata":{},"type":"1"},"txnMetadata":{"seqNo":3},"ver":"1"}
+{"reqSignature":{},"txn":{"data":{"alias":"John Jordan","dest":"iEvQcW7oFdsgNYJUwTRm6","role":"0","verkey":"~Srp7RBqd4PKqcdyWiigGYU"},"metadata":{},"type":"1"},"txnMetadata":{"seqNo":4},"ver":"1"}
+{"reqSignature":{},"txn":{"data":{"dest":"5pZSaMbNNTaarwjivRdz1p","role":"2","verkey":"3dNRSiSdxDxEvESBmW76azbUwSC8P4PeD655JXG8sS9m"},"metadata":{"from":"iEvQcW7oFdsgNYJUwTRm6"},"type":"1"},"txnMetadata":{"seqNo":5},"ver":"1"}
```

Comment:
Should we keep in the ReadMe file a list of provider (AWS, Azure, GGC, IBM, etc..) for each Stewards, just to have a quick look of the current network and keep in mind that we need diversity of provider when we add new node to the network? It's just a suggestion, maybe that is something we need to address with the Board eventually.